src/onboarding/index.html

```diff
@@ -23,46 +23,82 @@ <h2>Choose your search engine</h2>
+                <label for="search1" class="icon hidden"
+                  ><img src="./icons/GhosteryGlow-34x34.png"
+                /></label>
+                <input
+                  id="search1"
+                  type="radio"
+                  name="search"
+                  value="Ghostery Glow"
+                />
                 <label for="search1">Ghostery&nbsp;Glow</label>
               </li>
               <li>
-                <label for="search4" class="icon hidden"><img src="./icons/Bing-34x34.png" /></label>
+                <label for="search4" class="icon hidden"
+                  ><img src="./icons/Bing-34x34.png"
+                /></label>
                 <input id="search4" type="radio" name="search" value="Bing" />
                 <label for="search4">Bing</label>
               </li>
               <li>
-                <label for="search2" class="icon hidden"><img src="./icons/DuckDuckGo-34x34.png" /></label>
-                <input id="search2" type="radio" name="search" value="DuckDuckGo" />
+                <label for="search2" class="icon hidden"
+                  ><img src="./icons/DuckDuckGo-34x34.png"
+                /></label>
+                <input
+                  id="search2"
+                  type="radio"
+                  name="search"
+                  value="DuckDuckGo"
+                />
                 <label for="search2">DuckDuckGo</label>
               </li>
               <li>
-                <label for="search3" class="icon hidden"><img src="./icons/StartPage-34x34.png" /></label>
-                <input id="search3" type="radio" name="search" value="Startpage" />
+                <label for="search3" class="icon hidden"
+                  ><img src="./icons/StartPage-34x34.png"
+                /></label>
+                <input
+                  id="search3"
+                  type="radio"
+                  name="search"
+                  value="Startpage"
+                />
                 <label for="search3">Startpage</label>
               </li>
               <li>
-                <label for="search5" class="icon hidden"><img src="./icons/Google-34x34.png" /></label>
+                <label for="search5" class="icon hidden"
+                  ><img src="./icons/Google-34x34.png"
+                /></label>
                 <input id="search5" type="radio" name="search" value="Google" />
                 <label for="search5">Google</label>
               </li>
             </ul>
             <aside>
               <div class="hidden description" data-description-for="search1">
-                Ghostery Glow is built into Ghostery Dawn browser. This one-of-a-kind search engine does not log your search history, which means you get served objective results, not results that are filtered by the likelihood you’ll click on them.
+                Ghostery Glow is built into Ghostery Dawn browser. This
```

Comment:
can we extract those to a translation file? We should also update them 

@mdsandu can you please take a look?

Reply:
I don't argue with the purpose, but I would like to not extend the scope, and create another PR with the changes. However, I am not sure if it is possible to do with the static content in the HTML. The `chrome.i18n` is a dynamic JS API. Then, the only why would be to move to dynamic content generated for example in hybrids elements.

Reply:
#5